src/user_management/user.model.js

```diff
@@ -30,10 +30,10 @@ export default (sequelize, DataTypes) => {
     return bcrypt.compare(password, this.password)
   }
 
-  User.beforeCreate(async (user) => {
+  User.beforeCreate(async function(password) {
```

Comment:
### Did you run the linter?

Please run the linter inside docker, there is a spacing issue: `docker exec -ti tp-frontend npx standard --fix`